client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -468,7 +468,7 @@ export const ApplicationsTable: React.FC = () => {
           }) + "...",
         selectOptions: tagItems.map(({ name, tagName, categoryName }) => ({
           value: name,
-          label: name,
+          label: tagName,
```

Comment:
Humm, looks like this change is what is making the select box filter only on a tag name and not include the tag category.

Ref 1: 
https://github.com/konveyor/tackle2-ui/blob/8c8a0f852d861febebc48674b8aceac6b1c3911f/client/src/app/queries/tags.ts#L82-L92

Ref 2:
`filteredOptions()` up in the `MultiselectFilterControl`